apps/web/src/server/applications/pages/case/documentation/applications-documentation.controller.js

```diff
@@ -11,8 +15,44 @@ import { getCaseDocumentationCategories } from '../../../lib/services/case.servi
 export async function viewApplicationsCaseDocumentationCategories(request, response) {
 	const { caseId } = response.locals;
 	const documentationCategories = await getCaseDocumentationCategories(caseId);
-
 	const properties = { documentationCategories: sortBy(documentationCategories, ['displayOrder']) };
 
 	response.render(`applications/case/project-documentation`, properties);
 }
+
+/**
+ * View a folder, showing files in the folder, and listing subfolders
+ *
+ * @param {*} request
+ * @param {*} response
+ */
+export async function viewApplicationsCaseDocumentationFolder(request, response) {
+	const { caseId } = response.locals;
+	const { folderId } = request.params;
+	const topLevelDocumentationCategories = await getCaseDocumentationCategories(caseId);
+	const currentFolderProperties = {
+		topDocumentationCategories: sortBy(topLevelDocumentationCategories, ['displayOrder'])
+	};
+	const currentFileCategory = currentFolderProperties.topDocumentationCategories.find(
+		(cat) => cat.id === Number.parseInt(folderId, 10)
+	);
+
+	// TODO: get all the sub folders in this folder
+
+	/** @type {import('../../../applications.types').DocumentationPageProps }  */
+	const properties = {
+		caseId,
+		folderId,
+		folderName: currentFileCategory?.displayNameEn,
+		folderTree: null,
+		documentationFiles: null
+	};
+
+	// get the folderTree for breadcrumbing
+	properties.folderTree = getCaseDocumentationFolderTree(caseId, currentFileCategory);
```

Comment:
it's just a mock so it makes sense, but let's remember that these two are gonna be await functions